dynawo/sources/Models/CPP/ModelNetwork/DYNModelPhaseTapChanger.cpp

```diff
@@ -142,6 +121,6 @@ ModelPhaseTapChanger::evalZ(double t, state_g* g, ModelNetwork* network, double
       network->addEvent(id(), DYNTimeline(TapDown));
     }
   }
-}
+}  // ModelPhaseTapChanger::evalZ()
```

Comment:
Please remove ::evalZ

Reply:
Done, I removed the whole comment that was saying this } closes ::evalZ()